src/home/Footer.jsx

```diff
@@ -0,0 +1,118 @@
+              <Link style={{ textDecoration: "none" }}>
+                <li className="li">FAQ</li>
+              </Link>
+              <Link style={{ textDecoration: "none" }}>
+                <li className="li">Contact Us</li>
+              </Link>
+            </ul>
+          </div>
+          <div className="flexbox-item">
+            <h3 className="heading">Visit</h3>
+            <ul className="list">
+              <li className="li">3 Birrel Avenue</li>
+              <li className="li">Ashish</li>
+              <li className="li">Bangladesh, India</li>
+            </ul>
+          </div>
+          <div className="flexbox-item">
+            <h3 className="heading">Legal</h3>
+            <ul className="list">
+              <li className="li">
+                <a href="https://anitab.org/terms-of-use/" className="Links" target="_blank" rel="noopener noreferrer"  >
+                  Terms
+                </a>
+              </li>
+              <li className="li">
+                <a href="https://anitab.org/privacy-policy/" className="Links" target="_blank" rel="noopener noreferrer">
+                  Privacy
+                </a>
+              </li>
+            </ul>
+          </div>
+          <div className="flexbox-item">
+            <h3 className="heading">Get In Touch</h3>
+            <ul className="list">
+              <li className="li">
+                <a href="hello@bridgeintech.com" className="Links">
+                  hello@bridgeintech.com
+                </a>
+              </li>
+              <li className="li">+254713654608</li>
+            </ul>
+          </div>
+        </div>
+        <div className="bootomDiv">
+          <hr className="hr"></hr>
+          <div className="bottom-flexbox">
+            <div className="bottom-flexboxitem">
+              <h5 className="text-bottom">
+                Bridge In Tech. All Rights Reserved.
+              </h5>
+            </div>
+            <div className="bottom-flexboxitem">
+              <a href="https://github.com/anitab-org" target="_blank" rel="noopener noreferrer">
+                <img
+                  src={github}
+                  className="social-logos"
+                  alt="gihub_logo"
+                ></img>
+              </a>
+              <a href="https://www.linkedin.com/company/anitaborg-india/" target="_blank" rel="noopener noreferrer">
+                <img
+                  src={linkdin}
+                  className="social-logos"
+                  alt="linkdin_logo"
+                ></img>
+              </a>
+              <a href="https://twitter.com/anitab_org?lang=en" target="_blank" rel="noopener noreferrer">
+                <img
+                  src={twitter}
+                  className="social-logos"
+                  alt="twitter_logo"
+                ></img>
+              </a>
+              <a href="https://www.facebook.com/AnitaBorgIndia/" target="_blank" rel="noopener noreferrer">
```

Comment:
Why this is linked to `AntaBorgIndia` 's facebook page? It should be linked to AnitaB's [global facebook page](https://www.facebook.com/AnitaB.0rg/).